MAUI/LinearProgressBar/migration.md

```diff
@@ -321,3 +321,7 @@ private void LinearProgressBar_ProgressCompleted(object sender, ProgressValueEve
 </td>
 </tr>
 </table>
+
+## Unsupported feaures from Xamarin.Forms
+
+* In Xamarin, the [`Padding`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_Padding) is used to adjust padding of the progress and secondary progress at all the four (left, top, right and bottom) sides. But in .NET MAUI, it can be achieved with `ProgressPadding`, [`ProgressHeight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfLinearProgressBar.html#Syncfusion_Maui_ProgressBar_SfLinearProgressBar_ProgressHeight) and [`SecondaryProgressHeight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfLinearProgressBar.html?tabs=tabid-1#Syncfusion_Maui_ProgressBar_SfLinearProgressBar_SecondaryProgressHeight) properties. Please refer these `thickness` and `padding` documentation for details.
```

Comment:
* In Xamarin, the [`Padding`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.ProgressBar.SfLinearProgressBar.html#Syncfusion_XForms_ProgressBar_SfLinearProgressBar_Padding) is used to adjust the padding of the progress and secondary progress at all the four (left, top, right, and bottom) sides. But in .NET MAUI, it can be achieved with the `ProgressPadding,` [`ProgressHeight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfLinearProgressBar.html#Syncfusion_Maui_ProgressBar_SfLinearProgressBar_ProgressHeight) and [`SecondaryProgressHeight`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ProgressBar.SfLinearProgressBar.html?tabs=tabid-1#Syncfusion_Maui_ProgressBar_SfLinearProgressBar_SecondaryProgressHeight) properties. Please refer to these `thickness` and `padding` documentation for details.